setup.py

```diff
@@ -103,7 +103,7 @@ def find_package_data(
     return out
 
 
-PACKAGE = "django-notes"
+PACKAGE = "notes"
 NAME = "notes"
```

Comment:
I wonder if this was intended to be `django-notes`?

Reply:
Judging by https://pypi.python.org/pypi/django-notes/0.2.2 I'd say yes. Changed.